apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -296,175 +158,48 @@ private static Coverage transformPartA(MetricRegistry metricRegistry, Beneficiar
    *     Beneficiary}
    */
   private static Coverage transformPartB(MetricRegistry metricRegistry, Beneficiary beneficiary) {
-    Timer.Context timer =
```

Comment:
It appears that only `transformPartA` sets the Profile to `ProfileConstants.C4BB_COVERAGE_URL`

Reply:
thx...dumb mistake on my part...didn't even notice.

Reply:
Fixed.